sklearn/metrics/regression.py

```diff
@@ -573,3 +574,37 @@ def r2_score(y_true, y_pred, sample_weight=None,
         avg_weights = multioutput
 
     return np.average(output_scores, weights=avg_weights)
+
+
+def max_error(y_true, y_pred):
```

Comment:
I guess we can support sample_weight here, right?

Reply:
Ok. will add.

Reply:
See #3450 and #6217

Reply:
Thanks. I have made the change. Can you please review.